tests/core/test_snapshot_evaluator.py

```diff
@@ -156,16 +156,19 @@ def x(evaluator, y=None) -> None:
         partition_interval_unit=IntervalUnit.DAY,
         clustered_by=[],
         table_properties={},
+        table_description=None,
```

Comment:
Why do we need to specify `None` here? Is this something that can be set by default?

Reply:
I think this is because it gets passed explicitly in the chain of calls